docs/deployment/docker.md

```diff
@@ -16,6 +16,11 @@ are stateless, so for a production deployment you will want to set up and
 connect to an external PostgreSQL instance where the backend plugins can store
 their state, rather than using SQLite.
 
+> NOTE: If you do choose to use SQLite over the recommendation of using Postgres
+> then you will need to move `better-sqlite3` from the `devDependencies`
+> section to the `dependencies` section in the `\packages\backend\packages.json` file.
```

Comment:
```suggestion
> section to the `dependencies` section in the `packages/backend/package.json` file.
```

Windows user detected! :) But mainly it had an extra `s` in the file name.

Reply:
Thanks, funny thing is I've been on a MacBook for the last 6 months, old habits die hard I guess